soccer/gameplay/evaluation/ball.py

```diff
@@ -90,6 +90,8 @@ def angle_btw_three_pts(a, b, vertex):
         VA = math.sqrt((vertex.x - a.x)**2 + (vertex.y - a.y)**2)
         VB = math.sqrt((vertex.x - b.x)**2 + (vertex.y - b.y)**2)
         AB = math.sqrt((a.x - b.x)**2 + (a.y - b.y)**2)
+        if (VA * VA + VB * VB - AB * AB) / (2 * VA * VB) > 1:
```

Comment:
I don't know too much about this function, but if `VA` or `VB` are 0 this results in a division by 0. Are we 100% sure these will never be 0? (the same behavior is there below, I just want to check and bring it up).

Reply:
Could this be changed to use point.angle_between? This is confusing.

Reply:
It returns the angle between two directional vectors.